functional-tests/pom.xml

```diff
@@ -51,17 +51,20 @@
             <dependency>
                 <groupId>org.eclipse.che.selenium</groupId>
                 <artifactId>che-selenium-core</artifactId>
-                <version>${che.version}</version>
+                <version>6.14.0-SNAPSHOT</version>
+                <!--<version>${che.version}</version>-->
             </dependency>
             <dependency>
                 <groupId>org.eclipse.che.selenium</groupId>
                 <artifactId>che-selenium-test</artifactId>
-                <version>${che.version}</version>
```

Comment:
Is it just for testing purposes or you are going to merge it?

Reply:
That depends on whether we will be able to have those chances backported, or if we will have to wait until rh-che goes to 6.14.0

Reply:
I believe it would be better to wait. Making our build snapshot dependent does not sound like a good idea. If you need some particular changes from upstream you could also request a service release and we would update rh-che to `6.13.1` which would contain all changes required for this PR

Reply:
There was no plan from our side to depend on SNAPSHOT versions... For every change in upstream we depend on, we should wait for proper release which includes the change and then make relevant chagnes in our code.

We may (temporarily) break this rule for some extra urgent cases, but this one is not such case.

If I understand this PR correctly, it's just for collecting early feedback - and in that case, it's missing "WIP" label in description.